river/metrics/cluster/calinskiharabasz.py

```diff
@@ -0,0 +1,125 @@
+import math
+
+from river import stats, utils
+
+from . import base
+
+__all__ = ["CalinskiHarabasz"]
```

Comment:
I am not sure `__all__` is needed here. This is the only class in this file, right? I think the import in `__init__.py` already does the job. The same would apply to other similar cases

Reply:
You are right. `CalinskiHarabasz` is the only class in this file. I would proceed to delete all of them.

Reply:
Yes I'm not 100% sure on this. It does seem superfluous to have `__all__` when there's only one class.

Reply:
In this case I would proceed to delete all `__all__` where there is only one class!